packages/databyss-editor/lib/slateUtils.js

```diff
@@ -144,3 +144,37 @@ export const toggleMark = (editor, format) => {
     Editor.addMark(editor, format, true)
   }
 }
+
+// serialize slate node to html
+const serialize = node => {
+  if (Text.isText(node)) {
+    let _children = node.text
+    if (node.bold) {
+      _children = `<strong>${_children}</strong>`
+    }
+    if (node.italic) {
+      _children = `<i>${_children}</i>`
+    }
+    return _children
+  }
+
+  const children = node.children.map(n => serialize(n)).join('')
+
+  switch (node.type) {
```

Comment:
We will need to create a mockup and spec for the HTML export of sources and topics. We may even want to link them to routes in the app. Let's discuss next week.